specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/EasyAuthConfigs.json

```diff
@@ -0,0 +1,706 @@
+          "description": "The configuration settings of the Azure Active Directory token validation flow."
+        }
+      }
+    },
+    "AzureActiveDirectoryRegistration": {
+      "description": "The configuration settings of the Azure Active Directory app registration.",
+      "type": "object",
+      "properties": {
+        "openIdIssuer": {
+          "description": "The OpenID Connect Issuer URI that represents the entity which issues access tokens for this application.\nWhen using Azure Active Directory, this value is the URI of the directory tenant, e.g. https://login.microsoftonline.com/v2.0/{tenant-guid}/.\nThis URI is a case-sensitive identifier for the token issuer.\nMore information on OpenID Connect Discovery: http://openid.net/specs/openid-connect-discovery-1_0.html",
+          "type": "string"
+        },
+        "clientId": {
+          "description": "The Client ID of this relying party application, known as the client_id.\nThis setting is required for enabling OpenID Connection authentication with Azure Active Directory or \nother 3rd party OpenID Connect providers.\nMore information on OpenID Connect: http://openid.net/specs/openid-connect-core-1_0.html",
+          "type": "string"
+        },
+        "clientSecretRefName": {
+          "description": "The app secret ref name that contains the client secret of the relying party application.",
+          "type": "string"
+        },
+        "clientSecretCertificateThumbprint": {
+          "description": "An alternative to the client secret, that is the thumbprint of a certificate used for signing purposes. This property acts as\na replacement for the Client Secret. It is also optional.",
+          "type": "string"
+        },
+        "clientSecretCertificateSubjectAlternativeName": {
+          "description": "An alternative to the client secret thumbprint, that is the subject alternative name of a certificate used for signing purposes. This property acts as\na replacement for the Client Secret Certificate Thumbprint. It is also optional.",
+          "type": "string"
+        },
+        "clientSecretCertificateIssuer": {
+          "description": "An alternative to the client secret thumbprint, that is the issuer of a certificate used for signing purposes. This property acts as\na replacement for the Client Secret Certificate Thumbprint. It is also optional.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureActiveDirectoryLogin": {
+      "description": "The configuration settings of the Azure Active Directory login flow.",
+      "type": "object",
+      "properties": {
+        "loginParameters": {
+          "description": "Login parameters to send to the OpenID Connect authorization endpoint when\na user logs in. Each parameter must be in the form \"key=value\".",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "disableWwwAuthenticate": {
```

Comment:
>disableWwwAuthenticate [](http://example.com/codeflow?start=9&length=22)

Maybe something like authenticationMode: { includeWww, skipWww } ?

Reply:
Update as string enum

Reply:
Same comment using true or false in an enum does not make sense since there is no 3rd option then. Switch back to boolean or rename